src/index.d.ts

```diff
@@ -14,6 +14,30 @@ declare module 'bootstrap-vue' {
 
   export default VuePlugin
 
+  type TableVariant = 'active' | 'success' | 'info' | 'warning' | 'danger'
+  type SortDirection = 'asc' | 'desc'
+  type FormatterCallback = ((value: mixed, key: string, item: mixed) => string)
```

Comment:
@tmorehouse I know of instances where I use the formatter and return values that are not strings (which are converted to strings)
Is it wrong to force this to be a string return rather than `mixed` (or `any`)?

Reply:
Hmmm... yeah, that might be good. Not sure of the difference between `mixed` or `any`

Reply:
Probably `any` is safer than `mixed`, as it has a looser typing than `mixed`.